Automattic/wp-calypso
# Add new domain status pages navigation

#### Changes proposed in this Pull Request

Adds a separate component to handle the domain status navigation links. This is initial PR that will allow us to split the work in smaller PRs. We've decided to update the navigation now with the new component without feature-flagging it.

Here's how the navigation looks:

registered domain
<img width="749" alt="Screenshot 2020-03-25 at 15 10 41" src="https://user-images.githubusercontent.com/1355045/77542348-b51a8380-6eae-11ea-98fe-57da232b3b80.png">

mapped domain
<img width="743" alt="Screenshot 2020-03-25 at 15 10 48" src="https://user-images.githubusercontent.com/1355045/77542356-b9df3780-6eae-11ea-91d4-bab92a3f1247.png">

wpcom domain
<img width="739" alt="Screenshot 2020-03-25 at 15 10 55" src="https://user-images.githubusercontent.com/1355045/77542365-bd72be80-6eae-11ea-8dcc-00281d74fd02.png">

#### Testing instructions

* Check that all links in domain navigation are working
* Check that we're recording events for the wpcom domain links
